wgengine/watchdog.go

```diff
@@ -66,6 +96,18 @@ func (e *watchdogEngine) watchdogErr(name string, fn func() error) error {
 		buf := new(strings.Builder)
 		pprof.Lookup("goroutine").WriteTo(buf, 1)
 		e.logf("wgengine watchdog stacks:\n%s", buf.String())
+
+		// Collect the list of in-flight operations for debugging.
+		var b []byte
+		e.inFlightMu.Lock()
+		for k, t := range e.inFlight {
+			b = fmt.Appendf(b, "in-flight[%d]: name=%s start=%s\n", k.ctr, k.op, t.Format(time.RFC3339Nano))
```

Comment:
want to also print how long ago the operation was started? (which we usually round to a millisecond)